### PR TITLE
Update : Usage scp

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
I added how to copy the directory and how to use the port number in scp.

### CODE REVIEW

File: documentation/asciidoc/computers/remote-access/scp.adoc [hunk 1]
```diff
@@ -71,3 +71,21 @@ scp "my file.txt" pi@192.168.1.3:
...
+Copy the directory `project/` from your computer to the `pi` user's home folder of your Raspberry Pi at the IP address `192.168.1.3` with the following command:
+
+[,bash]
+----
+scp -r project/ pi@192.168.1.3:
+----
+
+=== Usage port number
+
+Copy the file to the `pi` user's home folder of your Raspberry Pi at the IP address `192.168.1.3:8001` with the following command(default `22`):
```
Comment (lurch) [inline documentation/asciidoc/computers/remote-access/scp.adoc:86]: IIRC we don't provide instructions for changing the SSH server port number, so it probably doesn't make sense to have instructions for connecting to a non-standard port? IMHO.

File: documentation/asciidoc/computers/remote-access/scp.adoc [hunk 2]
```diff
@@ -71,3 +71,21 @@ scp "my file.txt" pi@192.168.1.3:
 ----
 
 ======
+
+=== Copying Directory
```
[the remark this answers is not in the document]
Comment (RuffaloLavoisier) [reply documentation/asciidoc/computers/remote-access/scp.adoc]: @lurch Thank you. I reflected it.

File: documentation/asciidoc/computers/remote-access/scp.adoc (hunk 1 above)
Comment (RuffaloLavoisier) [reply documentation/asciidoc/computers/remote-access/scp.adoc:86]: @lurch Thank you for your opinion. I see. Let's listen to other people's opinions.

Comment (RuffaloLavoisier) [reply documentation/asciidoc/computers/remote-access/scp.adoc:86]: @lurch I looked it up. As you said, there is no information about port change. Should I take this part out? Or I will add a port change method to the <SSH> part. https://www.raspberrypi.org/documentation/computers/remote-access.html#ssh

Comment (lurch) [reply documentation/asciidoc/computers/remote-access/scp.adoc:86]: Probably better to take it out. The Raspberry Pi documentation is focused on the Raspberry-Pi-side of things, and isn't intended to be a definitive guide to every option of SSH and SCP, which can be found documented elsewhere on the internet :wink:

Comment (RuffaloLavoisier) [reply documentation/asciidoc/computers/remote-access/scp.adoc]: I reflected your opinion. Thank you for the good words.`The Raspberry Pi documentation is focused on the Raspberry-Pi-side of things`

**Review by lurch — APPROVED** [review]

**Comment by lurch** [comment]
Thank you for your contributions @RuffaloLavoisier

**Pull request merged**